### ISSUE TITLE
Bug - <sdds-dropdown> overlays sdds-navbar

### ISSUE THREAD

**Issue by blomqvist** [scania-digital-design-system/sdds#270]
**Describe the bug**  
`<sdds-dropdown>` in a `<table class="sdds-table sdds-table-compact sdds-table-divider sdds-on-white-bg">` have a higher z-index than `<nav class="sdds-navbar">`
![bild](https://user-images.githubusercontent.com/163365/148191356-e38f16d6-5025-4df6-9945-cd21a652830a.png)

**To Reproduce**   
Create navbar
```html
<nav class="sdds-navbar">
  <div class="sdds-navbar-overlay"></div>

  <button
    id="menu-mobile"
    class="sdds-navbar-icon-button sdds-navbar-side-menu-drawer"
    (click)="toggle()"
  >
    <span class="sdds-icon-drawer"></span>
  </button>

  <div class="sdds-navbar-application-brand">Application title</div>

  <div id="side-menu" class="sdds-navbar-collapsible {{ isExpanded ? 'expanded' : ''}}">
    <ul class="sdds-navbar-menu-list">
      <li
        class="sdds-navbar-menu-item"
        [routerLinkActive]="['link-active']"
        [routerLinkActiveOptions]="{ exact: true }"
      >
        <a class="sdds-navbar-menu-item-link" [routerLink]="['/']">Home</a>
      </li>
      <li class="sdds-navbar-menu-item" [routerLinkActive]="['link-active']">
        <a class="sdds-navbar-menu-item-link" [routerLink]="['/price-rows']"
          >Price rows</a
        >
      </li>
      <li class="sdds-navbar-menu-item" [routerLinkActive]="['link-active']">
        <a class="sdds-navbar-menu-item-link" [routerLink]="['/products']"
          >Products</a
        >
      </li>
      <li class="sdds-navbar-menu-item">
        <a class="sdds-navbar-menu-item-link" href="/swagger" target="_blank"
          >API (Swagger)</a
        >
      </li>
      <li class="sdds-navbar-menu-item" app-country></li>
    </ul>
  </div>
  <div class="sdds-navbar-scania-brand"></div>
</nav>

```

Create table
```html
<table class="sdds-table">
  <thead>
    <tr>
      <th>Test</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        <sdds-dropdown
          size="small"
          title="Vehicle state, new or used"
          name="vehicle-state"
          *ngIf="isNew"
          [defaultOption]="priceRow.vehicleState"
          (selectOption)="setVehicleState($event['detail'])"
        >
          <sdds-dropdown-option
            *ngFor="let opt of vehicleStateKeys"
            [value]="opt"
          >
            {{ opt }}
          </sdds-dropdown-option>
        </sdds-dropdown>
      </td>
    </tr>
  </tbody>
</table>
```

**Expected behavior**  
![bild](https://user-images.githubusercontent.com/163365/148192479-33159b8c-33bf-423e-bfb0-dcea2c893459.png)

**Screenshots**  
![bild](https://user-images.githubusercontent.com/163365/148192524-b2df4235-5173-4045-a947-503a01d1de8a.png)

**Version of SDDS or which package**  
```json
{
  "@scania/components": "^1.15.2",
  "@scania/icons": "^1.1.1",
  "@scania/theme-light": "^3.6.4"
}
```

**Application link**  
N/A

**System information:**  
Tested on macOS (latest) and firefox.

**Additional context**   
Setting `sdds-dropdown:host { z-index: 100; }` fixes the problem in this scenario. Might have other implications.

[AB#1358](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1358)

**Comment by mmexvr** [comment]
Thanks for the report!

**Comment by theJohnnyMe** [comment]
This should be fixed in component: 3.0.0.
